OracleTuxedo/tuxedo_full/README.md

```diff
@@ -0,0 +1,16 @@
+
+Tuxedo + SALT + TSAM Plus agent on Docker
+===============
+
+## How to run
+#Before running buildDockerImage.sh, you need to set the environment variables, http_proxy, https_proxy, ftp_proxy, and no_proxy, to access internet if the docker environment is behind a corporate proxy.
+
+The base image oracle/tuxedo:12.2.2 should be built before you run buildDockerImage.sh.
+Before getting started, download the Tuxedo 12.2.2 Linux 64 bit installer from http://www.oracle.com/technetwork/middleware/tuxedo/downloads/index.html
+Execute:
+buildDockerImage.sh
+
+You can then start the image in a new container with:  
+docker run -d -v \${LOCAL_DIR}:/u01/oracle/user_projects oracle/tuxedoall
+Note: \${LOCAL_DIR} is a local dir which used in docker image as external storage, it can be any dir.
```

Comment:
Please provide more instructions on what this `${LOCAL_DIR}` is used for as well as how to create it and how to set proper permissions so that the container can read and write to it.

Reply:
provided how to set permission to ${LOCAL_DIR}